vumi/tests/test_log.py

```diff
@@ -45,3 +46,19 @@ def test_error_log_levels(self):
             failure = entry['failure']
             exception = failure.trap(TracerException)
             self.assertEqual(exception, TracerException)
+
+
+class TestWrappingLogger(VumiTestCase):
+    def test_logging_methods(self):
+        log = WrappingLogger(system='test', bar='foo')
+
+        for method in (
+                log.debug, log.info, log.warning, log.error, log.critical,
+                log.msg, log.err):
+            with LogCatcher() as lc:
+                method('Test log')
+            [log] = lc.logs
+
+            self.assertEqual(log['system'], 'test')
+            self.assertEqual(log['bar'], 'foo')
```

Comment:
Should we perhaps also check that each method logs at the correct level? E.g. that `WrappingLogger.debug` actually sends a debug log message?